package.json

```diff
@@ -32,7 +32,8 @@
   "license": "MIT",
   "activationEvents": [
     "onLanguage:dart",
-    "workspaceContains:**/*.dart"
+    "workspaceContains:**/*.dart",
+    "onLan"
```

Comment:
questions: What is the "onLan" activationEvents? It's a typo error ?

Reply:
We used these changes for some months now. I made them back then and had stashed them to apply them on the latest version. 

Because of that stash and apply I can't see when I did this change. I guess I accidently added this and will remove it. Looke like there was something like "onLanguage:xxxx" like above.